Makefile

```diff
@@ -57,7 +57,7 @@ run-tests: ## Run CI tests
 	docker build --build-arg IMAGE_NAME=$(ASTRO_RUNTIME_IMAGE_NAME) -f dev/Dockerfile . -t astronomer-providers-dev
 	docker run -v `pwd`:/usr/local/airflow/astronomer_providers -v `pwd`/dev/.cache:/home/astro/.cache \
 	 	-w /usr/local/airflow/astronomer_providers \
-		--rm -it astronomer-providers-dev -- pytest tests
+		--rm -it astronomer-providers-dev -- pytest --cov astronomer --cov-report=term-missing tests
```

Comment:
Not much objection, however, would be nice to create a separate tiny PR for this. Because this change is unrelated with the title of this PR and if we want to trace the change in the commit history would be more visible if it has its own Title and maybe would be easy to revert that specific PR also if need be later.

Reply:
Yep, this is something I tested locally but forgot to remove 🤦‍♂️ I'll make another PR for it. I think this can be handy for locally testing!